_data/meltano/extractors/tap-spreadsheets-anywhere/ets.yml

```diff
@@ -50,10 +76,48 @@ settings:
         selected: true
     ```
 
-    See [repo README](https://github.com/ets/tap-spreadsheets-anywhere) for details.
+    See the `Common Config Examples` section below for more examples or see the [repo README](https://github.com/ets/tap-spreadsheets-anywhere) for more details.
   kind: array
   label: Tables
   name: tables
 settings_group_validation:
 - - tables
+usage: |
+  ## Common Config Examples
+
+  ### AWS Public IP Ranges JSON File
+
+  The JSON response from https://ip-ranges.amazonaws.com/ip-ranges.json can be parsed into records for each prefix in the `prefixes` array, using the `json_path` key.
+
+  ```yaml
+    config:
+      tables:
+      - path: https://ip-ranges.amazonaws.com
+        format: json
+        start_date: "2022-12-01T00:00:00Z"
+        key_properties: [ip_prefix]
+        name: aws_ips
+        pattern: "ip-ranges.json"
+        json_path: "prefixes"
+  ```
+
+  ### S3 JSONL files
+
+  ```
+    config:
+      tables:
+      - path: s3://<your-bucket-name>
+        format: json
+        key_properties: [user_id]
+        name: user_names
+        start_date: '2020-01-01T00:00:00Z'
+        pattern: <prefix>/user_names.json
```

Comment:
I think this will be hard to understand (that "pattern" refers to a regex pattern is already hard to grasp). I would put in an actual prefix use an actual example...

Reply:
@sbalnojan good point! I updated it to an example string instead of a placeholder.